stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -201,13 +201,18 @@ public class ChatClient internal constructor(
         return api.getGuestUser(userId, userName)
     }
 
+    @CheckResult
     public fun sendFile(
         channelType: String,
         channelId: String,
         file: File,
-        callback: ProgressCallback,
-    ) {
-        api.sendFile(channelType, channelId, file, callback)
+        callback: ProgressCallback? = null,
+    ): Call<String> {
+        return if (callback != null) {
+            api.sendFile(channelType, channelId, file, callback)
+        } else {
+            api.sendFile(channelType, channelId, file)
```

Comment:
We could probably merge those two methods in `ChatApi` so that we just call into a single one here

(But we shouldn't merge them in `FileUploader`, because it's public and Java clients couldn't make use of the default parameter value)